budgetApp1/budgetApp1/AuthenticationRepository.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  AuthenticationRepository.swift
+//  budgetApp1
+//
+//  Created by Lehlohonolo Nkadimeng on 2024/04/30.
+//
+
+import Foundation
```

Comment:
can remove foundations. UIKit imports it

Reply:
Done.